perun/collect/complexity/symbols.py

```diff
@@ -231,7 +231,10 @@ def _dismantle_symbols(symbol_map):
     specification_map = dict()
     for key in symbol_map.keys():
         # Process each symbol from the map
-        specification_map[key] = _process_symbol(symbol_map[key])
+        try:
+            specification_map[key] = _process_symbol(symbol_map[key])
+        except exceptions.UnexpectedPrototypeSyntaxError:
+            pass
```

Comment:
You sure it is good to catch this exception and ignore it? Maybe at least issue some warning?